business/istio_config.go

```diff
@@ -123,6 +124,49 @@ var newSecurityConfigTypes = []string{
 // GetIstioConfigList returns a list of Istio routing objects, Mixer Rules, (etc.)
 // per a given Namespace.
 func (in *IstioConfigService) GetIstioConfigList(ctx context.Context, criteria IstioConfigCriteria) (models.IstioConfigList, error) {
+	istioConfigList := models.IstioConfigList{}
+	// Check if user has access to the namespace (RBAC) in cache scenarios and/or
+	// if namespace is accessible from Kiali (Deployment.AccessibleNamespaces)
+	for cluster := range in.userClients {
+		singleClusterConfigList, err := in.GetIstioConfigListPerCluster(ctx, criteria, cluster)
+		if err != nil {
+			if api_errors.IsNotFound(err) || api_errors.IsForbidden(err) {
+				// If a cluster is not found or not accessible, then we skip it
+				log.Debugf("Error while accessing to cluster [%s]: %s", cluster, err.Error())
+				continue
+			}
+
+			if cluster == kubernetes.HomeClusterName && len(in.userClients) == 1 {
```

Comment:
Shouldn't be using the configuration name after this PR https://github.com/kiali/kiali/pull/6019 was merged?

`homeClusterName := config.Get().KubernetesConfig.ClusterName`

(I see some other uses like this in the code, also)

Reply:
I think we need a separate PR to remove all deprecated HomeClusterName, I added this line in a parallel with #6019.